tests/unit/test_ai_session.py

```
@@ -0,0 +1,188 @@
+		engine.backend.name = backend_name
+
+		def _search(query, limit=0):
+			if query.get("_type") == "ai" and "session_id" in query:
+				return prior_docs
+			return []
+		engine.search.side_effect = _search
+		task._get_query_engine = MagicMock(return_value=engine)
+		return task, engine
+
+	def test_fresh_when_no_prior_docs(self):
+		task, engine = self._make_task(prior_docs=[])
```

Ruff (0.15.18)

[warning] Unpacked variable `engine` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)